Description: Move server_dir path to /run
Author: Vincent Blut <vincent.debian@free.fr>
Forwarded: no
Last-Update: 2020-09-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/examples/chrony.nm-dispatcher.dhcp
+++ b/examples/chrony.nm-dispatcher.dhcp
@@ -11,7 +11,7 @@ action=$2
 
 chronyc=/usr/bin/chronyc
 default_server_options=iburst
-server_dir=/var/run/chrony-dhcp
+server_dir=/run/chrony-dhcp
 
 dhcp_server_file=$server_dir/$interface.sources
 # DHCP4_NTP_SERVERS is passed from DHCP options by NetworkManager.
